platform/openmote-cc2538/Makefile.openmote-cc2538

```diff
@@ -0,0 +1,50 @@
+# openmote-cc2538 platform makefile
+
+ifndef CONTIKI
+  $(error CONTIKI not defined! You must specify where CONTIKI resides!)
+endif
+
+### Configure the build for the board and pull in board-specific sources
+CONTIKI_TARGET_DIRS += . dev
+PLATFORM_ROOT_DIR = $(CONTIKI)/platform/$(TARGET)
+
+### Include
+CONTIKI_TARGET_SOURCEFILES += contiki-main.c board.c
+CONTIKI_TARGET_SOURCEFILES += leds-arch.c button-sensor.c openmote-sensors.c
+CONTIKI_TARGET_SOURCEFILES += antenna.c adxl346.c max44009.c sht21.c tps62730.c
+
+CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES)
+
+CLEAN += *.openmote-cc2538
+
+### Unless the example dictates otherwise, build with code size optimisations
+ifndef SMALL
+  SMALL = 1
+endif
+
+### Define the CPU directory
+CONTIKI_CPU=$(CONTIKI)/cpu/cc2538
+include $(CONTIKI_CPU)/Makefile.cc2538
+
+MODULES += core/net core/net/mac \
+           core/net/mac/contikimac \
+           core/net/llsec core/net/llsec/noncoresec
+
+PYTHON = python
+BSL_FLAGS += -e -w --bootloader-invert-lines -b 115200 -v
```

Comment:
It is not mandatory, but the new `cc2538-bsl` allows higher baudrates than 115200, for other platforms the `-b` argument was dropped